lib/vrl/stdlib/src/ip_ntoa.rs

```diff
@@ -0,0 +1,81 @@
+use std::{convert::TryInto, net::Ipv4Addr};
+
+use vrl::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct IpNtoa;
+
+impl Function for IpNtoa {
+    fn identifier(&self) -> &'static str {
+        "ip_ntoa"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            kind: kind::INTEGER,
+            required: true,
+        }]
+    }
+
+    fn examples(&self) -> &'static [Example] {
+        &[Example {
+            title: "Example",
+            source: r#"ip_ntoa!(67305985)"#,
+            result: Ok("1.2.3.4"),
+        }]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Compiled {
+        let value = arguments.required("value");
+
+        Ok(Box::new(IpNtoaFn { value }))
+    }
+}
+
+#[derive(Debug, Clone)]
+struct IpNtoaFn {
+    value: Box<dyn Expression>,
+}
+
+impl Expression for IpNtoaFn {
+    fn resolve(&self, ctx: &mut Context) -> Resolved {
+        let i: u32 = self
+            .value
+            .resolve(ctx)?
+            .try_integer()?
+            .try_into()
+            .map_err(|_| format!("cannot convert to bytes: integer does not fit in u32"))?;
+
+        let mut bytes = i.to_be_bytes();
+        bytes.reverse();
+        let ip = Ipv4Addr::from(bytes);
```

Comment:
This conversion will also be wrong on some platforms, I expect, due to the same `reverse` issue.

Reply:
Never mind, I see what this is doing.

Reply:
I _think_ I fixed this in https://github.com/timberio/vector/pull/8042/commits/08264214277ccddacf4c22307c8cfd1812b4a6cc